packages/account/package.json

```diff
@@ -31,7 +31,7 @@
     "dependencies": {
         "@binary-com/binary-document-uploader": "^2.4.8",
         "@deriv-com/analytics": "1.11.0",
-        "@deriv-com/translations": "1.3.5",
+        "@deriv-com/translations": "1.3.9",
```

Comment:
please elaborate why we are updating translations package in this PR?

Reply:
because Uz language first was added to the translation, so for translation package new npm version has been generated